languages/javascript/exercises/concept/array-destructuring/array-destructuring.js

```diff
@@ -0,0 +1,21 @@
+export function getFirstCard(deck) {
+  throw new Error('Implement the getFirstCard function')
+}
+
+export function getSecondCard(deck) {
```

Comment:
It's not a requirement, but if you're up for it, can we add a bit of JSDoc? The reason for this is that it will allow v3 in-browser coding and vscode/jetbrains IDE people to see type information! You don't *need* to do this. I'll create an improvement issue if you don't.

[Here is an example from another exercise.](https://github.com/exercism/v3/blob/master/languages/javascript/exercises/concept/array-analysis/array-analysis.js#L3-L10)

```javascript
/**
 * Get the position (index) of the card in the given stack
 *
 * @param {number[]} stack
 * @param {number} card
 *
 * @returns {number} position of the card in the stack
 */
export function getCardPosition(stack, card) {
  throw new Error('Implement the getCardPosition function')
}
```

Reply:
I'm not very familiar with TypeScript but I took a crack at this anyway. Hopefully I didn't end up making more work for you instead of less 🤞